src/query/storage/prom_converter.go

```diff
@@ -101,6 +103,7 @@ func toPromConcurrently(
 		mu       sync.Mutex
 	)
 
+	fastWorkerPool := readWorkerPool.Fast(100)
```

Comment:
nit: should we make this Java-like and more explicit what you get back here, i.e. make the method called`FastGoWithContextWorkerPool`? Otherwise it's not clear that only the `GoWithContext` is fast (and the other methods are the same).

Reply:
`FastGoWithContextWorkerPoolFactoryProviderBean` ? j/k will change.